public/app/features/live/centrifuge/remoteObservable.ts

```diff
@@ -0,0 +1,33 @@
+import * as comlink from 'comlink';
+import { from, Observable, switchMap } from 'rxjs';
+
+export const remoteObservableAsObservable = <T>(remoteObs: comlink.RemoteObject<Observable<T>>): Observable<T> =>
+  new Observable((subscriber) => {
+    // Passing the callbacks as 3 separate arguments is deprecated, but it's the only option for now
```

Comment:
This will be a problem only after we try to upgrade to RxJS 8 (which doesn't exist yet). I don't see a good solution except for creating an issue with this use case in their repo

Reply:
lets cross that bridge when we get there